diff --git a/vendor/go.opencensus.io/trace/trace.go b/vendor/go.opencensus.io/trace/trace.go
index 38ead7bf..8793af0a 100644
--- a/vendor/go.opencensus.io/trace/trace.go
+++ b/vendor/go.opencensus.io/trace/trace.go
@@ -261,6 +261,13 @@ func startSpanInternal(name string, hasParent bool, parent SpanContext, remotePa
 	return span
 }
 
+func (s *Span) SetSpanID(spanID SpanID) {
+	if s.data != nil {
+		s.data.SpanID = spanID
+	}
+	s.spanContext.SpanID = spanID
+}
+
 // End ends the span.
 func (s *Span) End() {
 	if s == nil {
